src/zspec/zdo/buffaloZdo.ts

```diff
@@ -1416,8 +1416,8 @@ export class BuffaloZdo extends Buffalo {
 
     //-- RESPONSES
 
-    public static readResponse(clusterId: ZdoClusterId, buffer: Buffer): unknown {
-        const buffalo = new BuffaloZdo(buffer, ZDO_MESSAGE_OVERHEAD); // set pos to skip `transaction sequence number`
+    public static readResponse(clusterId: ZdoClusterId, buffer: Buffer, skipZdoMessageOverhead: boolean): unknown {
```

Comment:
`hasZdoMessageOverhead` is better for the param name I think, since it's about whether it's there (emberznet) or not (zstack).

Reply:
Agree